### PR TITLE
Tags table config

### PR DESCRIPTION
**Pull request by mkubala**
Note: Base branch has been set to `tag-resolver-config` in order to not mix changes from the base branch (part of the PR #40).

Because of the number of files being changed I do recommend reviewing this PR commit-by-commit to make the process easier.

### CODE REVIEW

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala [hunk 1]
```diff
@@ -57,6 +57,22 @@ class SnapshotTableConfiguration(config: Config) {
...
+  private val cfg = config.asConfig("tables.tags.columnNames")
+  val name: String = cfg.asString("name", "name")
+
+  override def toString: String = s"TagsTableColumnNames($name)"
+}
+
+class TagsTableConfiguration(config: Config) {
+  private val cfg = config.asConfig("tables.tags")
+  val tableName: String = cfg.asString("tableName", "tags")
+  val schemaName: Option[String] = cfg.as[String]("schemaName").trim
```
Comment (devcateu) [inline core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:70]: I thinnk that you can use `cfg.asOptionalNonEmptyString`

Comment (devcateu) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:70]: or : `cfg.as("schemaName").trim`

File: core/src/main/scala/akka/persistence/postgres/tag/TagTables.scala [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+
   class EventTagTableDefinition(_tableTag: Tag)
-    extends Table[EventTag](
-      _tableTag,
-      _schemaName = None,
-      _tableName = "event_tag") {
+      extends Table[EventTag](_tableTag, _schemaName = tagsTableCfg.schemaName, _tableName = tagsTableCfg.tableName) {
     def * = (id, name) <> (EventTag.tupled, EventTag.unapply)
 
     val id: Rep[Int] = column[Int]("id", O.AutoInc)
```
Comment (devcateu) [inline core/src/main/scala/akka/persistence/postgres/tag/TagTables.scala:15]: I think that we can also add parameters for id column

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala (hunk 1 above)
Comment (mkubala) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:70]: 👍

File: core/src/main/scala/akka/persistence/postgres/tag/TagTables.scala (hunk 2 above)
Comment (mkubala) [reply core/src/main/scala/akka/persistence/postgres/tag/TagTables.scala:15]: 👍

**Pull request merged**